README.md

```diff
@@ -19,6 +19,9 @@ Only AWS users with access to your Key Management Store master key will be able
 ## Prerequisites to Using
 Before you can use the module, you need to have set in place several things:
 1. Install [the AWS CLI](http://docs.aws.amazon.com/cli/latest/userguide/installing.html) and [configure it](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) so that you can access your AWS account. Running `aws s3 ls` is a reasonable way to do this, assuming you are authorized to perform that operation.
+
+    * (Windows only) Set the AWS Region with the AWS_REGION environment variable
```

Comment:
Fixed. The reason here is aws-sdk uses AWS_REGION but the aws cli sets AWS_DEFAULT_REGION, causing the cli to fail even with a successful configure